deployment/command-line-option.md

```diff
@@ -161,21 +161,55 @@ Options:
 Control `fluentd` process using [Signals](signals.md) or Windows Event.
 
 ```text
-Usage: fluent-ctl COMMAND PID
+Usage: fluent-ctl COMMAND PID_OR_SVCNAME
 
 Commands:
   shutdown
   restart
   flush
   reload
+  dump
 ```
 
 ### Example
 
+You can specify the process id of the supervisor process.
+
 ```text
 fluent-ctl shutdown 11111
 ```
 
+If you run Fluentd as a Windows service, then you have to specify the svcname (the name of the Windows service).
+
+```text
+fluent-ctl dump fluentdwinsvc
+```
```

Comment:
The service name isn't mandatory required.
On windows, `fluentdwinsvc` is specified by default when the 3rd argument is omitted.

Reply:
Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/397/commits/dc0ddcfc33deb68577738093d0d519d23bf1c68f